wordpress-mobile/WordPress-Android
# Jetpack Section: Update FluxC for Activity Log Page Size

Parent #13629

This PR updates `FluxC` to that 'develop' version where `ActivityLogStore.ACTIVITY_LOG_PAGE_SIZE` is 100 (from previous 20). This step is required in order in order to avoid some of the scrolling issues on the 'Backups' screen due to the fact that the API enhancement is not ready yet.

To test:
- Launch app and go to `My Site` tab.
- Click on the `Activity log` menu item to launch the `Activity Log` screen.
- Scroll to the bottom and verify that the pagination works as expected and that this time the page size is larger (100, from previous 20).
- Go back and click on the `Backup` menu item to launch the `Backup` screen.
- Again, scroll to the bottom and verify that the pagination works as expected and that this time the page size is larger (100, from previous 20).
- If you previously encountered an issue with the `Backup` screen where there were no more items to load, due to the fact that the there were only a couple of rewindable items, thus not filling the available screen estate, verify that this problem is now gone or at least that it became less relevant.

PR submission checklist:
